### PR TITLE
HS-108 Implement Add bedroom view

### PR DESCRIPTION
**Pull request by minhho3199**
Full UI for add bedroom view but missing tests

### CODE REVIEW

File: lib/theme/hs_color.dart [hunk 1]
```diff
@@ -26,6 +26,7 @@ class HSColor {
   static const Color statusSuccess = Color(0xFF34C759);
   static const Color statusInformational = Color(0xFF007AFF);
 
+  static const Color neutral9 = Color(0xFF141414);
```
Comment (suesitran) [inline lib/theme/hs_color.dart:29]: please add comment 'new design' so we know what value has been updated with new design

**Review by suesitran — CHANGES_REQUESTED** [review]

File: lib/features/add_bedroom/view/add_bedroom_counter.dart [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class AddBedroomCounter extends StatelessWidget {
+  final ValueNotifier<int>? counter;
+  final String? text;
+  const AddBedroomCounter(
+      {super.key, required this.counter, required this.text});
```
Comment (suesitran) [inline lib/features/add_bedroom/view/add_bedroom_counter.dart:9]: why is `counter` nullable? why is `text` nullable? they're required field, so if they need to be nullable, there must be a usecase where null value is used. What is that usecase?

File: lib/features/add_bedroom/view/add_bedroom_counter.dart [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(bottom: 24),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Text(
+            text!,
```
Comment (suesitran) [inline lib/features/add_bedroom/view/add_bedroom_counter.dart:28]: without null check, app may crash here if `text` value is null

**Review by suesitran — CHANGES_REQUESTED** [review]

File: lib/features/add_bedroom/view/add_bedroom_counter.dart (hunk 2 above)
Comment (minhho3199) [reply lib/features/add_bedroom/view/add_bedroom_counter.dart:9]: That's my bad. I was sort of blindly following old code for the buttons and just did it this way

File: lib/features/add_bedroom/view/add_bedroom_counter.dart (hunk 3 above)
Comment (minhho3199) [reply lib/features/add_bedroom/view/add_bedroom_counter.dart:28]: I removed the nullables so a check here would not be necessary I believe

File: lib/l10n/intl_en.arb [hunk 4]
```diff
@@ -27,6 +27,12 @@
   "signinErrorToastMessage": "Unable to sign you in at the moment. Please try again later.",
   "signinErrorToastTitle":"Login Failed",
   "loadingMessage": "Loading...",
+  "addBedroomViewInstructions": "How many bedrooms, bathrooms, parking?",
+  "bedroomText": "Bedrooms",
+  "bathroomText": "Bathrooms",
+  "parkingText": "Parkings",
+  "back": "Back",
+  "next": "Next",
```
Comment (suesitran) [inline lib/l10n/intl_en.arb:35]: Please remove this, @dlvinh  will add in his PR

**Review by suesitran — APPROVED** [review]

**Pull request merged**